src/sql/src/plan/statement/ddl.rs

```diff
@@ -3761,6 +3765,9 @@ fn iceberg_sink_builder(
     let Some(namespace) = namespace else {
         sql_bail!("Iceberg sink must specify NAMESPACE");
     };
+    if commit_interval.is_none() {
+        sql_bail!("Iceberg sink must specify COMMIT INTERVAL");
+    }
```

Comment:
Is there a reason to force this option?

Reply:
For now yes, there is no default. In the near future after benchmarking but before beta we will pick a default